pkg/api/types.go

```diff
@@ -720,6 +720,9 @@ type GitRepoVolumeSource struct {
 // as files using the keys in the Data field as the file names.
 // Secret volumes support ownership management and SELinux relabeling.
 type SecretVolumeSource struct {
+	// Specify whether the Secret must be defined
+	// +optional
+	Optional *bool
```

Comment:
I was told to order them as they logically appear. If Optional makes sense at the end I can move it.